frame/staking/src/lib.rs

```diff
@@ -2019,10 +2019,25 @@ decl_module! {
 				<Error<T>>::NoUnlockChunk
 			);
 
+			let active_ring = ledger.active_ring;
+			let active_kton = ledger.active_kton;
+
 			ledger.rebond(plan_to_rebond_ring, plan_to_rebond_kton);
 
 			Self::update_ledger(&controller, &mut ledger);
 
+			let rebond_ring = active_ring.saturating_sub(ledger.active_ring);
+			let rebond_kton = active_kton.saturating_sub(ledger.active_kton);
```

Comment:
```suggestion
			let rebond_ring = ledger.active_ring.saturating_sub(active_ring);
			let rebond_kton = ledger.active_kton.saturating_sub(active_kton);
```

I think should be like this way?

Reply:
Ah yes.